### PR TITLE
LRDOCS-5001, LRDOCS-5002, LRDOCS-5003, Managing User Data article drafts

### PR DESCRIPTION
**Pull request by rbohl**
Resending Pull Request. https://issues.liferay.com/browse/LRDOCS-5001

### CODE REVIEW

File: discover/portal/articles/190-managing-users/03-managing-user-data/01-sanitizing-user-data.markdown [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+Once you've worked your way through the items and taken action, the view in the
+Application Data Review screen is updated to reflect that there ore no more items
+needed to review. Click the *Complete Step* button once finished.
+
+![Figure 7: Click *Complete Step* once all of the User's application data is reviewed and dealt with.](../../../images/users-data-review-complete.png)
+
+## Step 4: Anonymize Remaining Data [](id=step-4-anonymize-remaining-data)
+
+In step 4, you must click *Anonymize Data*. This completes the anonymization
```
[the remark this answers is not in the document]
Comment (rbohl) [reply discover/portal/articles/190-managing-users/03-managing-user-data/01-sanitizing-user-data.markdown]: @dejuknow 
Sorry Dennis, so considering my reply to the last comment about not mentioning the cases that could arise if there are dev customizations, do we need to change anything here?

**Comment by rbohl** [comment]
@dejuknow I merged master and made three commits incorporating your suggestions: 1 commit per article. Please review the commits and let me know if these are ready to go or need more edits. 

See my two comments regarding dev customizations, too.

File: discover/portal/articles/190-managing-users/03-managing-user-data/00-intro.markdown [hunk 2]
```diff
@@ -36,13 +39,17 @@ $$$
...
 
-Anonymizing User Associated Data (UAD) is all about identity swapping. A User
-being anonymized must have their identifiers (for example, User ID and their
-User Name) removed from content they've interacted with. However, portal content
-usually requires this information for its applications to work properly.
-Therefore, the User's identifiers must be replaced by something, or someone.
-Meet the new User, *Anonymous Anonymous*. This deactivated User is dedicated to
-be the User whose identifiers are assigned to anonymized content. 
+Deleting User data is the safest way to honor _right to be forgotten_ requests.
+When Userr data must be preserved, automatic anonymization of the data is in
```
Comment (dejuknow) [inline discover/portal/articles/190-managing-users/03-managing-user-data/00-intro.markdown]: Typo: Userr -> User

Would also change, "automatic anonymization of the data is in order" to just "anonymization of the data is in order"

File: discover/portal/articles/190-managing-users/03-managing-user-data/01-sanitizing-user-data.markdown (hunk 1 above)
[the remark this answers is not in the document]
Comment (dejuknow) [reply discover/portal/articles/190-managing-users/03-managing-user-data/01-sanitizing-user-data.markdown]: That's fine for now. Perhaps once the dev docs are in, we can add references that explain how this behavior might differ from the default.

**Comment by dejuknow** [comment]
Made one edit and replied to comments.

**Comment by rbohl** [comment]
Fixed the typo, and I agree, we can point to the developer docs when written, to point out the possible customizations to the default behavior. Closing, forwarded here: https://github.com/sez11a/liferay-docs/pull/3243

**Pull request closed without merging**